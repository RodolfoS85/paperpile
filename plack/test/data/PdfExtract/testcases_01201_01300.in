---
file: ../../../PDFREPOSITORY/001201.pdf
---
file: ../../../PDFREPOSITORY/001202.pdf
---
file: ../../../PDFREPOSITORY/001203.pdf
---
file: ../../../PDFREPOSITORY/001204.pdf
---
file: ../../../PDFREPOSITORY/001205.pdf
---
file: ../../../PDFREPOSITORY/001206.pdf
---
file: ../../../PDFREPOSITORY/001207.pdf
---
file: ../../../PDFREPOSITORY/001208.pdf
---
file: ../../../PDFREPOSITORY/001209.pdf
---
file: ../../../PDFREPOSITORY/001210.pdf
---
file: ../../../PDFREPOSITORY/001211.pdf
---
file: ../../../PDFREPOSITORY/001212.pdf
---
file: ../../../PDFREPOSITORY/001213.pdf
---
file: ../../../PDFREPOSITORY/001214.pdf
---
file: ../../../PDFREPOSITORY/001215.pdf
---
file: ../../../PDFREPOSITORY/001216.pdf
---
file: ../../../PDFREPOSITORY/001217.pdf
---
file: ../../../PDFREPOSITORY/001218.pdf
---
file: ../../../PDFREPOSITORY/001219.pdf
---
file: ../../../PDFREPOSITORY/001220.pdf
---
file: ../../../PDFREPOSITORY/001221.pdf
---
file: ../../../PDFREPOSITORY/001222.pdf
---
file: ../../../PDFREPOSITORY/001223.pdf
---
file: ../../../PDFREPOSITORY/001224.pdf
---
file: ../../../PDFREPOSITORY/001225.pdf
---
file: ../../../PDFREPOSITORY/001226.pdf
---
file: ../../../PDFREPOSITORY/001227.pdf
---
file: ../../../PDFREPOSITORY/001228.pdf
---
file: ../../../PDFREPOSITORY/001229.pdf
---
file: ../../../PDFREPOSITORY/001230.pdf
---
file: ../../../PDFREPOSITORY/001231.pdf
---
file: ../../../PDFREPOSITORY/001232.pdf
---
file: ../../../PDFREPOSITORY/001233.pdf
---
file: ../../../PDFREPOSITORY/001234.pdf
---
file: ../../../PDFREPOSITORY/001235.pdf
---
file: ../../../PDFREPOSITORY/001236.pdf
---
file: ../../../PDFREPOSITORY/001237.pdf
---
file: ../../../PDFREPOSITORY/001238.pdf
---
file: ../../../PDFREPOSITORY/001239.pdf
---
file: ../../../PDFREPOSITORY/001240.pdf
---
file: ../../../PDFREPOSITORY/001241.pdf
---
file: ../../../PDFREPOSITORY/001242.pdf
---
file: ../../../PDFREPOSITORY/001243.pdf
---
file: ../../../PDFREPOSITORY/001244.pdf
---
file: ../../../PDFREPOSITORY/001245.pdf
---
file: ../../../PDFREPOSITORY/001246.pdf
---
file: ../../../PDFREPOSITORY/001247.pdf
---
file: ../../../PDFREPOSITORY/001248.pdf
---
file: ../../../PDFREPOSITORY/001249.pdf
---
file: ../../../PDFREPOSITORY/001250.pdf
---
file: ../../../PDFREPOSITORY/001251.pdf
---
file: ../../../PDFREPOSITORY/001252.pdf
---
file: ../../../PDFREPOSITORY/001253.pdf
---
file: ../../../PDFREPOSITORY/001254.pdf
---
file: ../../../PDFREPOSITORY/001255.pdf
---
file: ../../../PDFREPOSITORY/001256.pdf
---
file: ../../../PDFREPOSITORY/001257.pdf
---
file: ../../../PDFREPOSITORY/001258.pdf
---
file: ../../../PDFREPOSITORY/001259.pdf
---
file: ../../../PDFREPOSITORY/001260.pdf
---
file: ../../../PDFREPOSITORY/001261.pdf
---
file: ../../../PDFREPOSITORY/001262.pdf
---
file: ../../../PDFREPOSITORY/001263.pdf
---
file: ../../../PDFREPOSITORY/001264.pdf
---
file: ../../../PDFREPOSITORY/001265.pdf
---
file: ../../../PDFREPOSITORY/001266.pdf
---
file: ../../../PDFREPOSITORY/001267.pdf
---
file: ../../../PDFREPOSITORY/001268.pdf
---
file: ../../../PDFREPOSITORY/001269.pdf
---
file: ../../../PDFREPOSITORY/001270.pdf
---
file: ../../../PDFREPOSITORY/001271.pdf
---
file: ../../../PDFREPOSITORY/001272.pdf
---
file: ../../../PDFREPOSITORY/001273.pdf
---
file: ../../../PDFREPOSITORY/001274.pdf
---
file: ../../../PDFREPOSITORY/001275.pdf
---
file: ../../../PDFREPOSITORY/001276.pdf
---
file: ../../../PDFREPOSITORY/001277.pdf
---
file: ../../../PDFREPOSITORY/001278.pdf
---
file: ../../../PDFREPOSITORY/001279.pdf
---
file: ../../../PDFREPOSITORY/001280.pdf
---
file: ../../../PDFREPOSITORY/001281.pdf
---
file: ../../../PDFREPOSITORY/001282.pdf
---
file: ../../../PDFREPOSITORY/001283.pdf
---
file: ../../../PDFREPOSITORY/001284.pdf
---
file: ../../../PDFREPOSITORY/001285.pdf
---
file: ../../../PDFREPOSITORY/001286.pdf
---
file: ../../../PDFREPOSITORY/001287.pdf
---
file: ../../../PDFREPOSITORY/001288.pdf
---
file: ../../../PDFREPOSITORY/001289.pdf
---
file: ../../../PDFREPOSITORY/001290.pdf
---
file: ../../../PDFREPOSITORY/001291.pdf
---
file: ../../../PDFREPOSITORY/001292.pdf
---
file: ../../../PDFREPOSITORY/001293.pdf
---
file: ../../../PDFREPOSITORY/001294.pdf
---
file: ../../../PDFREPOSITORY/001295.pdf
---
file: ../../../PDFREPOSITORY/001296.pdf
---
file: ../../../PDFREPOSITORY/001297.pdf
---
file: ../../../PDFREPOSITORY/001298.pdf
---
file: ../../../PDFREPOSITORY/001299.pdf
---
file: ../../../PDFREPOSITORY/001300.pdf
