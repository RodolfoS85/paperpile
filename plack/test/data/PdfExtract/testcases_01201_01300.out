---
authors: 'Seet, Bruce T. and Dikic, Ivan and Zhou, Ming-Ming and Pawson, Tony'
doi: 10.1038/nrm1960
title: Reading protein modifications with interaction domains
---
authors: 'Segal, Eran and Fondufe-Mittendorf, Yvonne and Chen, Lingyi and Thastrom, Ann Christine and Field, Yair and Moore, Irene K. and Wang, Ji-Ping Z. and Widom, Jonathan'
doi: 10.1038/nature04979
title: A genomic code for nucleosome positioning
---
authors: 'Segal, Eran and Shapira, Michael and Regev, Aviv and Peer, Dana and Botstein, David and Koller, Daphne and Friedman, Nir'
title: 'Module networks: identifying regulatory modules and their condition-specific regulators from gene expression data'
---
authors: 'Seiler, Markus and Mehrle, Alexander and Poustka, Annemarie and Wiemann, Stefan'
doi: 10.1186/1471-2105-7-144
title: The 3of5 web application for complex and comprehensive pattern matching in protein sequences
---
authors: 'Selbach, Matthias and Schwanhausser, Bjorn and Thierfelder, Nadine and Fang, Zhuo and Khanin, Raya and Rajewsky, Nikolaus'
doi: 10.1038/nature07228
title: Widespread changes in protein synthesis induced by microRNAs
---
authors: 'Selmer, M. and -D. Su, X.'
title: Crystal structure of an mRNA-binding fragment of Moorella thermoacetica elongation factor SelB
---
authors: 'Semsey, Szabolcs and Benjamin, Julie-Anna M. and Mitarai, Namiko and Krishna, Sandeep and Csiszovszki, Zsolt and Sneppen, Kim and Massé, Eric'
title: Dynamics of the recovery from sRNA-mediated gene silencing
---
authors: 'Sengupta, Anirvan M. and Djordjevic, Marko and Shraiman, Boris I.'
doi: 10.1073/pnas.022388499
title: Specificity and robustness in transcription control networks
---
authors: 'Serganov, Alexander and Patel, Dinshaw J.'
doi: 10.1038/nrg2172
title: 'Ribozymes, riboswitches and beyond: regulation of gene expression without proteins'
---
authors: 'Serpico, Sebastiano B. and Bruzzone, Lorenzo'
title: A NEW SEARCH ALGORITHM FOR FEATURE SELECTION IN HYPERSPECTRAL REMOTE SENSING IMAGES
---
authors: 'Shakhnovich, Boris E. and Deeds, Eric and Delisi, Charles and Shakhnovich, Eugene'
title: Protein Structure and Evolutionary History Determine Sequence Space Topology
---
authors: 'Shao, YU and Chan, Chi YU and Maliyekkel, Anil and Lawrence, Charles E. and Roninson, Igor B. and Ding, YE'
doi: 10.1261/rna.546207
title: Effect of target secondary structure on RNAi efficiency
---
authors: 'Sharan, R.'
title: Cluster Analysis and its Applications to Gene Expression Data
---
authors: 'Sharma, Cynthia M. and Hoffmann, Steve and Darfeuille, Fabien and Reignier, Jeremy and Findeiß, Sven and Sittka, Alexandra and Chabas, Sandrine and Reiche, Kristin and Hackermuller, Jorg and Reinhardt, Richard and Stadler, Peter F. and Vogel, Jorg'
doi: 10.1038/nature08756
title: The primary transcriptome of the major human pathogen Helicobacter pylori
---
authors: 'Shen, Haiou and Zhang, Hantao'
title: Improving Exact Algorithms for MAX-2-SAT
---
authors: 'Shen-Orr, Shai S. and Milo, Ron and Mangan, Shmoolik and Alon, Uri'
doi: 10.1038/ng881
title: Network motifs in the transcriptional regulation network of Escherichia coli
---
authors: 'Shen, Ling X. and Basilion, James P. and Stanton, Jr., Vincent P.'
title: Single-nucleotide polymorphisms can cause different structural folds of mRNA
---
authors: 'Sherry, S. T. and -H. Ward, M. and Kholodov, M. and Baker, J. and Phan, L. and Smigielski, E. M. and Sirotkin, K.'
title: 'dbSNP: the NCBI database of genetic variation'
---
authors: 'Shevchenko, Andrej and Sunyaev, Shamil and Loboda, Alexander and Shevchenko, Anna and Bork, Peer and Ens, Werner and Standing, Kenneth G.'
doi: 10.1021/ac0013709
title: Charting the Proteomes of Organisms with Unsequenced Genomes by MALDI-Quadrupole Time-of-Flight Mass Spectrometry and BLAST Homology Searching
---
authors: 'Shigenobu, Yoshimitsu and Del Carpio, Carlos A.'
title: 'A Bioinformatic Approach for RNA 3D Structure Prediction: Development of a Knowledge-Base for 2D-to-3D Structural Elements Compatibility Analysis'
---
authors: 'Shimizu, Akio and Tada, Kouichirou and Shukunami, Chisa and Hiraki, Yuji and Kurokawa, Tsutomu and Magane, Noriko and Kurokawa-Seo, Misuzu'
doi: 10.1074/jbc.M003535200
title: A Novel Alternatively Spliced Fibroblast Growth Factor Receptor 3 Isoform Lacking the Acid Box Domain Is Expressed during Chondrogenic Differentiation of ATDC5 Cells
---
authors: 'Shmulevich, Ilya and Dougherty, Edward R. and Kim, Seungchan and Zhang, Wei'
title: 'Probabilistic Boolean networks: a rule-based uncertainty model for gene regulatory networks'
---
authors: 'Shmygelska, Alena and Hoos, Holger H'
doi: 10.1186/1471-2105-6-30
title: An ant colony optimisation algorithm for the 2D and 3D hydrophobic polar protein folding problem
---
authors: 'Shoemaker, D. D. and Schadt, E. E. and Armour, C. D. and He, Y. D. and Garrett-Engele, P. and McDonagh, P. D. and Loerch, P. M. and Leonardson, A. and Lum, P. Y. and Cavet, G. and Wu, L. F. and Altschuler, S. J. and Edwards, S. and King, J. and Tsang, J. S. and Schimmack, G. and Schelter, J. M. and Koch, J. and Ziman, M. and Marton, M. J. and Li, B. and Cundiff, P. and Ward, T. and Castle, J. and Krolewski, M. and Meyer, M. R. and Mao, M. and Burchard, J. and Kidd, M. J. and Dai, H. and Phillips, J. W. and Linsley, P. S. and Stoughton, R. and Scherer, S. and Boguski, M. S.'
title: Experimental annotation of the human genome using microarray technology
---
authors: 'Shubin, Neil H.'
doi: 10.1002/jmor.10017
title: 'Origin of Evolutionary Novelty: Examples From Limbs'
---
authors: 'Shubin, Neil and Tabin, Cliff and Carroll, Sean'
title: 'Fossils, genes and the evolution of animal limbs'
---
authors: 'Shui, Xiuqi and McFail-Isom, Lori and Hu, Gary G. and Williams, Loren Dean'
title: The B-DNA Dodecamer at High Resolution Reveals a Spine of Water on Sodium
---
authors: 'a, Paolo Sibani and van der Pasb, Ruud and Christian Schiin c, J.'
title: The lid method for exhaustive exploration of metastable states of complex systems
---
authors: 'Siebert, Sven and Backofen, Rolf'
title: Methods for Multiple Alignment and Consensus Structure Prediction of RNAs implemented in MARNA
---
authors: 'Siebert, Sven and Backofen, Rolf'
doi: 10.1089/cmb.2006/0089
title: A Dynamic Programming Approach for Finding Common Patterns in RNAs
---
authors: 'Sikorski, Andrzej and Kolinski, Andrzej and Skolnick, Jeffrey'
title: Computer simulations of protein folding with a small number of distance restraints
---
authors: 'Sikorski, Andrzej and Romiszowski, Piotr'
title: Monte Carlo simulations of protein-like heteropolymers
---
authors: 'Simorre, Jean-Pierre and Legault, Pascale and Hangar, Arlene B. and Michiels, Paul and Pardi, Arthur'
title: A Conformational Change in the Catalytic Core of the Hammerhead Ribozyme upon Cleavage of an RNA Substrate
---
authors: 'Singh, Natalia N. and Androphy, Elliot J. and Singh, Ravindra N.'
doi: 10.1261/rna.7580704
title: In vivo selection reveals combinatorial controls that define a critical exon in the spinal muscular atrophy genes
---
authors: 'Singh, Amit P.'
title: 'Protein Structure Alignment: A Comparison of Methods'
---
authors: 'Singh, Amit P. and Brutlag, Douglas L.'
title: Hierarchical Protein Structure Superposition using both Secondary Structure and Atomic Representations
---
authors: 'Singh, Rohit and Xu, Jinbo and Berger, Bonnie'
doi: 10.1073/pnas.0806627105
title: Global alignment of multiple protein interaction networks with application to functional orthology detection
---
authors: 'Singh, Rohit and Xu, Jinbo and Berger, Bonnie'
title: GLOBAL ALIGNMENT OF MULTIPLE PROTEIN INTERACTION NETWORKS
---
authors: 'Sinha, Saurabh and Blanchette, Mathieu and Tompa, Martin'
doi: 10.1186/1471-2105-5-170
title: 'PhyME: A probabilistic algorithm for finding motifs in sets of orthologous sequences'
---
authors: 'Sinha, Rileen and Hiller, Michael and Pudimat, Rainer and Gausmann, Ulrike and Platzer, Matthias and Backofen, Rolf'
doi: 10.1186/1471-2105-9-477
title: Improved identification of conserved cassette exons using Bayesian networks
---
authors: 'Sitbon, Einat and Pietrokovski, Shmuel'
doi: 10.1186/1472-6807-7-3
title: Occurrence of protein structure elements in conserved sequence regions
---
authors: 'Sittka, Alexandra and Lucchini, Sacha and Papenfort, Kai and Sharma, Cynthia M. and Rolle, Katarzyna and Binnewies, Tim T. and Hinton, Jay C. D. and Vogel, Jorg'
doi: 10.1371/journal.pgen.1000163
title: 'Deep Sequencing Analysis of Small Noncoding RNA and mRNA Targets of the Global Post-Transcriptional Regulator, Hfq'
---
authors: 'Sittka, Alexandra and Sharma, Cynthia M. and Rolle, Katarzyna and Vogel, Jörg'
title: Deep sequencing of Salmonella RNA associated with heterologous Hfq proteins in vivo reveals small RNAs as a major target class and identifies RNA processing phenotypes
---
authors: 'Skolnick, Jeffrey and Kolinski, Andrzej and Ortiz, Angel R.'
title: 'MONSSTER: A Method for Folding Globular Proteins with a Small Number of Distance Restraints'
---
authors: 'Smalheiser, Neil R. and Torvik, Vetle I.'
title: Complications in Mammalian microRNA Target Prediction
---
authors: 'Smit, Arian F. A. and Toth, Gabor and Riggs, Arthur D. and Jurka, Jerzy'
title: 'Ancestral, Mammalian-wide Subfamilies of LINE-1 Repetitive Sequences'
---
authors: 'Smith, Temple'
title: Comparison of Biosequences
---
authors: 'Smith, Christopher W. J. and Valcárcel, Juan'
title: 'Alternative pre-mRNA splicing: the logic of combinatorial control'
---
authors: 'Soen, Yoav and Mori, Akiko and Palmer, Theo D and Brown, Patrick O'
doi: 10.1038/msb4100076
title: Exploring the regulation of human neural precursor cell differentiation using arrays of signaling microenvironments
---
authors: 'Sol, Ricard V. and Fernndez, Pau and Kauffman, Stuart A.'
title: 'Adaptive walks in a gene network model of morphogenesis: insights into the Cambrian explosion'
---
authors: 'Somol, Petr and Pudil, Pavel'
title: Fast Branch & Bound Algorithm in Feature Selection
---
authors: 'Song, Yinglei'
title: Tree Decomposition Based Fast Search of RNA Structures Including Pseudoknots in Genomes
---
authors: 'Song, Dandan and Deng, Zhidong'
title: A Fuzzy Dynamic Programming Approach to Predict RNA Secondary Structure
---
authors: 'Song, Tianyan and Mika, Franziska and Lindmark, Barbro and Liu, Zhi and Schild, Stefan and Bishop, Anne and Zhu, Jun and Camilli, Andrew and Johansson, Jörgen and Vogel, Jörg'
doi: 10.1111/j.1365-2958.2008.06392.x
title: A new Vibrio cholerae sRNA modulates colonization and affects release of outer membrane vesicles
---
authors: 'Song, Guang and Thomas, Shawna and Dill, Ken A. and MARTIN SCHOLTZ, J. and Amato, Nancy M.'
title: A PATH PLANNING-BASED STUDY OF PROTEIN FOLDING WITH A CASE STUDY OF HAIRPIN FORMATION IN PROTEIN G AND L
---
authors: 'Sonnhammer, Erik L. L. and von Heijne, Gunnar and Krogh, Anders'
title: A hidden Markov model for predicting transmembrane helices in protein sequences
---
authors: 'Sonnleitner, Elisabeth and Sorger-Domenigg, Theresa and Madej, Monika J. and Findeiss, Sven and Hackermuller, Jorg and Huttenhofer, Alexander and Stadler, Peter F. and Blasi, Udo and Moll, Isabella'
doi: 10.1099/mic.0.2008/019703-0
title: Detection of small RNAs in Pseudomonas aeruginosa by RNomics and structure-based bioinformatic tools
---
authors: 'Soper, Toby J. and Woodson, Sarah A.'
doi: 10.1261/rna.1110608
title: The rpoS mRNA leader recruits Hfq to facilitate annealing with DsrA sRNA
---
authors: 'Sorek, Rotem and Ast, Gil and Graur, Dan'
doi: 10.1101/gr.229302
title: Alu-Containing Exons are Alternatively Spliced
---
authors: 'Sorek, Rotem and Ast, Gil'
doi: 10.1101/gr.1208803
title: Intronic Sequences Flanking Alternatively Spliced Exons Are Conserved Between Human and Mouse
---
authors: 'Sorek, Rotem and Dror, Gideon and Shamir, Ron'
doi: 10.1186/1471-2164-7-273
title: Assessing the number of ancestral alternatively spliced exons in the human genome
---
authors: 'Sorek, Rotem and Shemesh, Ronen and Cohen, Yuval and Basechess, Ortal and Ast, Gil and Shamir, Ron'
doi: 10.1101/gr.2572604
title: A Non-EST-Based Method for Exon-Skipping Prediction
---
authors: 'Soukup, Garrett A. and Derose, Elizabeth C. and Koizumi, Makoto and Breaker, Ronald R.'
title: Generating new ligand-binding RNAs by affinity maturation and disintegration of allosteric ribozymes
---
authors: 'Southall, Noel T. and Dill, Ken A. and Haymet, A. D. J.'
doi: 10.1021/jp015514e
title: A View of the Hydrophobic Effect
---
authors: 'Souza, Augustine E. and Göringer, H. Ulrich'
title: The guide RNA database
---
authors: 'Spang, Rainer and Vingron, Martin'
title: Statistics of large-scale sequence searching
---
authors: 'di Fenizio, Pietro Speroni and Banzhaf, Wolfgang'
title: Stability of Metabolic and Balanced Organisations
---
authors: 'Sperschneider, Jana and Datta, Amitava'
doi: 10.1261/rna.968808
title: 'KnotSeeker: Heuristic pseudoknot detection in long RNA sequences'
---
authors: 'ARIA GRAZIA SPILLANTINI, M and Murrell, Jill R. and Goedert, Michel and Farlow, M Artin R. and Lug, Aaron K and Ghetti, Bernardino'
title: Mutation in the tau gene in familial multiple system tauopathy with presenile dementia
---
authors: 'Spitz, Francois and Salminen, Marjo and Demignon, Josiane and Kahn, Axel and Daegelen, Dominique and Maire, Pascal'
title: A Combination of MEF3 and NFI Proteins Activates Transcription in a Subset of Fast-Twitch Muscles
---
authors: 'St Johnston, Daniel'
doi: 10.1038/nrm1643
title: 'MOVING MESSAGES: THE INTRACELLULAR LOCALIZATION OF mRNAs'
---
authors: 'Stadler, Peter F.'
title: Fitness Landscapes
---
authors: 'Stadler, Michael B. and Shomron, Noam and Yeo, Gene W. and Schneider, Aniket and Xiao, Xinshu and Burge, Christopher B.'
doi: 10.1371/journal.pgen.0020191
title: Inference of Splicing Regulatory Activities by Sequence Neighborhood Analysis
---
authors: 'Stadler, Barbel M. R. and Stadler, Peter F. and Wagner, Gunter P.'
title: 'The topology of the possible: Formal spaces underlying patterns of evolutionary change'
---
authors: 'Staffa, Alfredo and Acheson, Nicholas H. and Cochrane, Alan'
title: Novel Exonic Elements That Modulate Splicing of the Human Fibronectin EDA Exon
---
authors: 'Stahl, Yvonne and Simon, RÜDIGER'
doi: 10.1387/ijdb.041929ys
title: Plant stem cell niches
---
authors: 'Stamm, Stefan'
title: 'Signals and their transduction pathways regulating alternative splicing: a new dimension of the human genome'
---
authors: 'Staple, David W. and Butcher, Samuel E.'
doi: 10.1371/journal.pbio.0030213.g001
title: 'Pseudoknots: RNA Structures with Diverse Functions'
---
authors: 'Stark, Alexander and Brennecke, Julius and Russell, Robert B. and Cohen, Stephen M.'
doi: 10.1371/journal.pbio.0000060
title: Identification of Drosophila MicroRNA Targets
---
authors: 'Steglich, Claudia and Futschik, Matthias E. and Lindell, Debbie and Voss, Bjoern and Chisholm, Sallie W. and Hess, Wolfgang R.'
doi: 10.1371/journal.pgen.1000173
title: 'The Challenge of Regulation in a Minimal Photoautotroph: Non-Coding RNAs in Prochlorococcus'
---
authors: 'Steglich, Claudia and Futschik, Matthias and Rector, Trent and Steen, Robert and Chisholm, Sallie W.'
doi: 10.1128/JB.01097-06
title: Genome-Wide Analysis of Light Sensing in Prochlorococcus
---
authors: 'Steigele, Stephan and Huber, Wolfgang and Stocsits, Claudia and Stadler, Peter F and Nieselt, Kay'
doi: 10.1186/1741-7007-5-25
title: Comparative analysis of structured RNAs in S. cerevisiae indicates a multitude of different functions
---
authors: 'Steinhfel, K. and Skaliotis, A. and Albrecht, A. A.'
title: Stochastic Protein Folding Simulation in the d-Dimensional HP-Model
---
authors: 'Stewart, Jennifer J. and Stargell, Laurie A.'
doi: 10.1074/jbc.M105276200
title: The Stability of the TFIIA-TBP-DNA Complex Is Dependent on the Sequence of the TATAAA Element
---
authors: 'Thesis, Bachelor'
title: 3D-Structure-Motifs Aware Sequence Structure Alignment of RNAs
---
authors: 'Liu, Ding and Panigrahy, Rina and Lehman, Eric and Charikar, Moses'
title: 'Approximating the Smallest Grammar: Kolmogorov Complexity in Natural Models'
---
authors: 'Stoeckert, Jr., Christian J. and Salas, Fidel and Brunk, Brian and Overton, G. Christian'
title: 'EpoDB: a prototype database for the analysis of genes expressed during vertebrate erythropoiesis'
---
authors: 'Stoesser, Guenter and Tuli, Mary Ann and Lopez, Rodrigo and Sterk, Peter'
title: The EMBL Nucleotide Sequence Database
---
authors: 'Storey, John D.'
title: A direct approach to false discovery rates
---
authors: 'Torey, John D. S'
title: 'THE POSITIVE FALSE DISCOVERY RATE: A BAYESIAN INTERPRETATION AND THE q-VALUE'
---
authors: 'Storey, John D. and Tibshirani, Robert'
doi: 10.1073/pnas.1530509100
title: Statistical significance for genomewide studies
---
authors: 'Stormo, Gary D.'
title: 'DNA binding sites: representation and discovery'
---
authors: 'Stoye, Jens'
title: Multiple sequence alignment with the divide-and-conquer method
---
authors: 'Sturla, L-M and Merrick, AE and Burchill, SA'
doi: 10.1038/sj.bjc.6601249
title: 'FGFR3IIIS: a novel soluble FGFR3 spliced variant that modulates growth is frequently expressed in tumour cells'
---
authors: 'Sturm, Nancy R. and Campbell, David A.'
title: The Role of Intron Structures in trans-Splicing and Cap 4 Formation for the Leishmania Spliced Leader RNA
---
authors: 'Sugimoto, Naoki and Nakano, Shu-ich and Yoneyama, Mari and Honda, Kei-ich'
title: Improved thermodynamic parameters and helix initiation factor to predict stability of DNA duplexes
---
authors: 'Sugnet, Charles W. and Srinivasan, Karpagam and Clark, Tyson A. and OBrien, Georgeann and Cline, Melissa S. and Wang, Hui and Williams, Alan and Kulp, David and Blume, John E. and Haussler, David and Ares, Jr., Manuel'
doi: 10.1371/journal.pcbi.0020004.eor
title: Unusual Intron Conservation near Tissue-Regulated Exons Found by Splicing Microarrays
---
authors: 'Sun, Hanzhen and Chasin, Lawrence A.'
title: Multiple Splicing Defects in an Intronic False Exon
---
authors: 'Sun, Lele and Cui, Zhiyong and Gottlieb, Robert L. and Zhang, Biliang'
title: A Selected Ribozyme Catalyzing Diverse Dipeptide Synthesis
---
authors: 'Sun, Ying-Fei and Fan, Xiao-Dan and Li, Yan-Da'
title: 'Identifying splicing sites in eukaryotic RNA: support vector machine approach'
